index.js

```diff
@@ -5,9 +5,23 @@ const fs = require('fs')
 const http = require('http')
 const url = require('url')
 const shortid = require('shortid')
+const formidable = require('formidable')
+require('dotenv').config()
 
 const ipaddr = require('./ipaddr')
 
+
+const UPLOAD_DIR = process.env.UPLOAD_DIR || './tmp'
```

Comment:
This is not the standard temporary directory. I just realised the problem is it's different on Windows vs Mac/Linux (/tmp/). We should use `os.tmpdir()`